### ISSUE TITLE
Home screen shows waiting spindle after installation.

### ISSUE THREAD

**Issue by kaul-vineet** [microsoft/Power-CAT-Copilot-Studio-Kit#37]
### Home screen shows waiting spindle after installation. 

When installation is complete (solution import), and user goes to the Home Page, the page loads with a waiting **spindle**. This is displayed till first test run is executed. Possibly looking for data after installation. 

![image](https://github.com/user-attachments/assets/7fbda631-c975-474f-bf2e-c5662b0e376f)

**Comment by gulzarudheen** [comment]
@kaul-vineet I attempted to reproduce the issue, and the page doesn't require an initial test run to load the homepage, it does take around 1 to 2 minutes to load on the initial attempt after import.

Loaded without any test run: 
<img width="931" alt="image" src="https://github.com/user-attachments/assets/7c7ba5d0-08b1-41c6-9302-2bb4407514a3">

**Comment by gulzarudheen** [comment]
@kaul-vineet I’d like to clarify that the behavior you’ve noticed regarding the loading time of the custom page within the model-driven app is expected and not specific to the page you’re working with. It’s normal for the initial load after a solution import to take few minutes, but this delay typically does not occur in subsequent loads. This behavior can be observed with other custom pages as well.
When you embed a custom page into a model-driven app, the platform needs to load various resources to render the custom page properly. I recommend raising a support ticket to address this product performance issue further.

**Comment by MunozVictor** [comment]
Hi, I'm trying to configure and test Copilot Studio Kit for SharePoint synchronization. I installed it, but the options menu is not showing:

![Image](https://github.com/user-attachments/assets/2f49534e-da32-462f-9b37-e3938a9d8a9f)

I followed the instructions (I tried installing it from the store and using the .zip file) and enabled all cloud flows. (Development environment)

Anyone able to help me?
